### ISSUE TITLE
Specifying queue explicitly in create_newcase fails

### ISSUE THREAD

**Issue by jayeshkrishna** [ESMCI/cime#1483]
Specifying a queue to create_newcase using the "--queue=<QUEUE_NAME>" 
option fails with the following error message,
```
Traceback (most recent call last):
  File "./create_newcase", line 219, in <module>
    _main_func(__doc__)
  File "./create_newcase", line 202, in _main_func
    input_dir=input_dir)
  File "/lcrc/group/acme/jayesh/ESMCI_cime/scripts/Tools/../../scripts/lib/CIME/case.py", line 744, in configure
    env_batch.set_job_defaults(bjobs, pesize=maxval, walltime=walltime, force_queue=queue, allow_walltime_override=test)
  File "/lcrc/group/acme/jayesh/ESMCI_cime/scripts/Tools/../../scripts/lib/CIME/XML/env_batch.py", line 227, in set_job_defaults
    self.set_value("JOB_QUEUE", queue, subgroup=job)
UnboundLocalError: local variable 'queue' referenced before assignment

```

**Comment by jayeshkrishna** [comment]
This is a missing assignment.
I am working on this and a PR is coming up.

**Comment by jayeshkrishna** [comment]
This was fixed in another recent commit -  7a67baee4ed71621520f6e8c039bbf2b8b3c872b
